### PR TITLE
SchedulerClient creates a new login session instead of renewing an existing one

### PR DESCRIPTION
**Pull request by lpellegr**

### CODE REVIEW

File: rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SessionStore.java [hunk 1]
```diff
@@ -115,4 +118,19 @@ public int terminateExpiredSessions(long timeoutDelay) {
...
     }
+
+    public String renewSession(String oldSessionId) throws NotConnectedException {
+        Session oldSession = sessions.remove(oldSessionId);
+
+        if (oldSession == null) {
+            return oldSessionId;
+        }
+
+        String newSessionId = SessionIdGenerator.newSessionId();
```
Comment (fviale) [inline rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SessionStore.java]: I don't understand why a new sessionId must be created as we are renewing the lease of an existing rest session

**Comment by lpellegr** [comment]
@fviale Could you please have a look?

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
@fviale @marcocast Could you please review the changes?

File: rest/rest-server/src/test/java/org/ow2/proactive_grid_cloud_portal/common/SessionTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import org.junit.Test;
+import org.ow2.proactive.scheduler.common.exception.NotConnectedException;
+import org.ow2.proactive.scheduler.common.util.SchedulerProxyUserInterface;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class SessionTest {
+
+    @Test
```
Comment (fviale) [inline rest/rest-server/src/test/java/org/ow2/proactive_grid_cloud_portal/common/SessionTest.java]: a little description ?

**Comment by marcocast** [comment]
looks ok to me.

**Pull request merged**